frictionless/schema/schema.py

```diff
@@ -262,6 +262,10 @@ def to_excel_template(self, path: str):
 
     def to_summary(self) -> str:
         """Summary of the schema in table format"""
+        # Setting required property of the field which is included as custom fields
```

Comment:
So we need to remove these 4 lines

Reply:
Now I understand, I will remove these lines. I should have checked the standards. Sure adding "validation error" would be better. I will add that.